ui/tests/acceptance/clients/counts-test.js

```diff
@@ -29,10 +29,6 @@ module('Acceptance | clients | counts', function (hooks) {
     return authPage.login();
   });
 
-  hooks.after(function () {
```

Comment:
👀 is there a reason we don't need to remove the `sinon.stub(timestsamp, 'now')` in the `before` hook here and in `ui/tests/acceptance/clients/counts/acme-test.js`, while we _do_ need to remove it in `ui/tests/acceptance/clients/counts/overview-test.js`?

Reply:
I was trying to figure this out! From what I can tell - I don't think this stub is actually necessary for this test 😆

Reply:
Yep - it looks like many of these tests don't actually need the timestamped stubbed. I've removed it from tests where I'm confident that's the case, otherwise I've left it in case the timestamp util is called somewhere during the acceptance/integration flow of the test so I don't accidentally introduce a weird future test failure